xformers/triton/k_softmax.py

```diff
@@ -81,17 +79,11 @@ def _softmax(
 
     if use_mask:
         mask_ptrs = M + n * stride_mn + k
-        add_mask = tl.load(mask_ptrs, io_mask, other=float("-inf"))
+        add_mask = tl.load(mask_ptrs, io_mask, other=float("-inf")).to(tl.float32)
         x += add_mask
 
     # compute numerically-stable softmax
     z = x - tl.max(x, axis=0)
-
-    if is_fp16:
```

Comment:
not required anymore, computations inside the kernel are fp32 and that's all fine and dandy